tests/run-make/incr-prev-body-beyond-eof/Makefile

```diff
@@ -1,19 +0,0 @@
-# ignore-none no-std is not supported
-# ignore-nvptx64-nvidia-cuda FIXME: can't find crate for `std`
```

Comment:
Problem: I would prefer keeping this, because at least there's a fixme explaining why it was ignored